spec/controllers/api/logs_controller_spec.rb

```diff
@@ -122,13 +122,14 @@
     context "when attempting to destroy one's own log" do
       let(:user) { log.user }
 
-      it 'destroys the log' do
-        expect { delete_destroy }.to change { Log.find_by(id: log.id) }.from(Log).to(nil)
-      end
+      context 'when the log has more than one entry' do
+        before { expect(log.log_entries.size).to be > 1 }
 
-      it 'returns a 204 status code' do
-        delete_destroy
-        expect(response).to have_http_status(204)
+        it 'destroys the log and returns a 204 status code without raising an N+1 query error' do
+          expect { delete_destroy }.to change { Log.find_by(id: log.id) }.from(Log).to(nil)
+
+          expect(response).to have_http_status(204)
+        end
```

Comment:
Here I am combining two examples into one. I used to be somewhat religious, at least in unit specs, about having only one expectation per example. However, in feature specs, for performance reasons, we combine lots of expectations into a single example, and really it doesn't seem to cause much (or really almost any) difficulty. So, I think I'm going to probably start combining some more unit tests together, too, when it feels appropriate. This seems to me like a decent occasion for consolidation.